src/apps/competitions/unpackers/codabench_v1.py

```diff
@@ -0,0 +1,76 @@
+
+from django.utils import timezone
+
+from competitions.unpackers.v2 import V2Unpacker
+from .utils import CompetitionUnpackingException
+from ..models import Competition
+
+
+class CodabenchV1Unpacker(V2Unpacker):
```

Comment:
I don't think we need a whole new unpacker for codabench? Will have to discuss with Isabelle a bit more?

Reply:
okay, that's a good idea